packages/core/src/DdSdkReactNative.tsx

```diff
@@ -212,6 +211,13 @@ export class DdSdkReactNative {
         return DdSdk.clearAllData();
     };
 
+    /**
+     * Returns `true` if the SDK was initialized, `false` otherwise.
+     */
+    public static wasInitialized() {
+        return DdSdkReactNative._wasInitialized;
+    }
```

Comment:
Two notes about this one:

- I declared it as a getter at first, but it was more cumbersome to mock in tests, and I figured it would be acceptable in this form as well - I am open to change my mind.
- I have considered exposing it as `_wasInitialized()` since the only reason I have made this change is to be able to use it internally. However, it could be useful to our users as well, even though it is just a simple static property which is set to `true` after the initialization has been completed.